media/js/mozorg/contact-spaces.js

```diff
@@ -115,12 +115,16 @@
             //set the current tab navigation item
             $('ul.category-tabs li[data-id="' + tab + '"]').addClass('current');
 
-            // set the current list menu navigation item
-            $('.nav-category li[data-id="' + id + '"]').addClass('current');
+            if (tab === 'spaces') {
+                // set the current list menu navigation item for the active space
+                $('.nav-category li[data-id="' + id + '"]').addClass('current');
+            }
 
-            // hide the community sub menu's
-            $('.accordion .submenu').hide();
-            mozMap.toggleCommunitySubMenu();
+            if (tab === 'communities') {
+                // hide the community sub menu's
+                $('.accordion .submenu').hide();
```

Comment:
I know the logic isn't changed in this PR, but I feel this line would be clearer if targeted at the `nav-communities` ID, as it appears the intent is to affect a specific element and not all `.accordion` elements. Thoughts?

Also, looks like a conditional around these two `communities` related lines should have been there before this PR, as they shouldn't need to fire for `spaces` or `contact`. :+1: